tests/DCPS/KeyTest/.gitignore

```diff
@@ -61,3 +61,6 @@
 /BoundTest2TypeSupportS.h
 /BoundTest2TypeSupportS.inl
 /IsBounded
+/invalid_*TypeSupport.idl
+/invalid_*TypeSupportImpl.cpp
+/invalid_*TypeSupportImpl.h
```

Comment:
I would rather have these ignores generated by the script, but maybe that's a discussion for another time / another PR.